builder/googlecompute/driver_mock.go

```diff
@@ -117,17 +119,23 @@ func (d *DriverMock) CreateImage(name, description, family, zone, disk string, i
 	if d.CreateImageResultSizeGb == 0 {
 		d.CreateImageResultSizeGb = 10
 	}
-
+	imageFeatures := make([]*compute.GuestOsFeature, len(image_features))
```

Comment:
```suggestion
	imageFeatures := make([]*compute.GuestOsFeature, len(image_features), 0)
```

Reply:
Hello,

Thank you for your review! Please let me fix it!

Reply:
Hello,

Please review 3fd0bb0:

    * Fixed GuestOsFeatures list
    * Added test to verify that the slice is correctly created.

Thank you!